### PR TITLE
Fix TileType logs

### LINKED ISSUE #2591: "to place a 2"
image

### PR DESCRIPTION
**Pull request by daniel-ziegler**
Fixes #2591, this time client-side

### CODE REVIEW

File: src/components/LogPanel.ts [hunk 1]
```diff
@@ -95,6 +96,9 @@ export const LogPanel = Vue.component('log-panel', {
             manifest.standardActions,
           ]);
           if (card && card.cardType) return this.parseCardType(card.cardType, data.value);
+        } else if (data.type === LogMessageDataType.TILE_TYPE) {
```
Comment (kberg) [inline src/components/LogPanel.ts:99]: see the transalatableMessageDataType structure above? Keep that up to date also, please.

Comment (daniel-ziegler) [reply src/components/LogPanel.ts:100]: Updated. (Doesn't change the logic given the way the code is structured.)

Should `LogMessageDataType.CARD` also be added, then?

Comment (kberg) [reply src/components/LogPanel.ts:100]: Good question, I'll chat separately with @alrusdi about that.

**Review by kberg — APPROVED** [review]
Thank you!

**Pull request merged**